lib/stdlib/src/array.erl

```diff
@@ -14,73 +14,14 @@
 %%
 %% @author Richard Carlsson <carlsson.richard@gmail.com>
 %% @author Dan Gudmundsson <dgud@erix.ericsson.se>
```

Comment:
Should the edoc `@author` tags be converted into moduledoc attributes as well?

Reply:
Yes, done. So many details, it is hard to get everything correct :)